src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const nav = document.querySelector('.gallery__list');
+const imgLarge = document.querySelector('.gallery__large-img');
+
+nav.addEventListener('click', (events) => {
+  events.preventDefault();
+
+  const link = events.target.closest('.gallery__thumb');
+
+  if (!link) {
+    return;
+  }
+
+  imgLarge.src = link.src;
```

Comment:
The `link.src` might not be correct if `link` is not an image element. Ensure that you are accessing the correct attribute to update `imgLarge.src`.